### ISSUE TITLE
Decimals For MVP

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#5656]
## Description + Acceptance Criteria

Ignoring #5649 for the MVP (too many API changes for an MVP) 

TLDR: input and display should be the same, regardless of how we convert and store as integer.

MVP behaviors for "Edit/Create Group Page Requirement Form Field inputs": 
For ALL requirement forms:
  - The Amount input should accept decimals as the user expects to use them.
  - Amount inputs will have helper text beneath them (details next)

- [x] For ERC20 Requirement, the helper text under the Amount input should read "Using 18 decimal precision".
- [x] For ERC721 Requirement, there is NO helper text.
  - [ ] Should not accept decimals, only integers in the Amount input.
- [x] For Cosmos Base Token Requirement, the helper text under the Amount input should read "Using 6 decimal precision".
- [x] For EVM Base Token Requirement, the helper text under the Amount input should read "Using 18 decimal precision".

[Helper text should follow design specifications in the Design System for Form Field Inputs](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=339-5968&mode=design&t=QficsQYKcVVXjZdt-0)

MVP Behavior for the "Members Page, Groups Tab": 
- For displaying the "Amount" in the Requirement display, we must convert the integer to default decimal for each requirement type:
 
- [x] ERC20, 18 decimal precision: the integer "1000000000000000000" should be input + displayed as "1"
- [x] ERC721, 0 decimal precision: the integer "1" should be input + displayed as "1"
- [x] EVM Base Token, 18 decimal precision: the integer "1005000000000000000" should be input +  displayed as "1.005"
- [x] Cosmos Base Token, 6 decimal precision: the integer "1000000" should be input +  displayed as "1". 

## Project Owner

@zakhap 

## Additional context

Example of how the helper text should look, based on the Input in the Component Kit: 

<img width="576" alt="Screen Shot 2023-11-09 at 11 20 52 AM" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/65cccc1b-3a8c-4d9e-89e0-d1f3d52b5243">
